src/wp-includes/fonts/class-wp-font-collection.php

```diff
@@ -0,0 +1,260 @@
+<?php
+/**
+ * Font Collection class.
+ *
+ * This file contains the Font Collection class definition.
+ *
+ * @package    WordPress
+ * @subpackage Font Library
```

Comment:
The subpackage is Fonts:

```suggestion
 * @subpackage Fonts
```

Same change for each of the new files in the `wp-includes/fonts/` directory.